alphabase/psm_reader/maxquant_reader.py

```diff
@@ -11,6 +12,8 @@
     psm_reader_yaml,
 )
 
+warnings.filterwarnings("always")
```

Comment:
could the scope of this be reduced, e.g. to a specific code block and/or specific warnings?

Reply:
Maybe you know more about this.
This code is needed to enforce that warning are shown because it's somehow not happening.
So it's not removing warnings but showing them.

Reply:
ah, I misinterprested this as "filter always" .. then it's fine, maybe add the information that warnings are otherwise not shown as a comment ..